prow/cmd/branchprotector/protect.go

```diff
@@ -35,40 +35,31 @@ import (
 type options struct {
 	config    string
 	jobConfig string
-	token     string
 	confirm   bool
-	endpoint  flagutil.Strings
+	github    cmdutil.GitHubOptions
 }
 
 func (o *options) Validate() error {
-	if o.config == "" {
-		return errors.New("empty --config-path")
-	}
-
-	if o.token == "" {
-		return errors.New("empty --github-token-path")
+	o.github.TokenRequired = true
```

Comment:
When is this not true?

Reply:
> When is this not true?

Never.  But not all of the commands ad the default token path before this PR.  This setting is me waffling about whether moving them to the newly-shared default `/etc/github/oauth` is a breaking change ;).

Reply:
Changes to defaults for these flags are breaking, yes

Reply:
> Changes to defaults for these flags are breaking, yes

The affected commands are going from "emptybdefault, error to unset" to "default to `etc/github/oauth`".  So existing callers must be setting theboption and shouldn't be affected.  Ne callers now might get the default; I'm not sure how gracefully these commands handle broken token paths, but I doubt it's worse than dying with "you need to set the flag".  Anything you want me to dig into more?

Reply:
Oh, I see -- a move from failing on empty to defaulting on empty is OK

Reply:
So we can get rid of `TokenRequired` now correct?

Reply:
> So we can get rid of `TokenRequired` now correct?

Sure.  Dropped in 804ad9c -> 3953c0e.  Now I check against empty values all the time to protect against crazy people explicitly setting `-github-token-path=`, but I can drop that as well if you like.